dynamo/plot/utils.py

```diff
@@ -505,9 +519,10 @@ def _matplotlib_points(
                 thresh=0,
                 levels=100,
             )
-            ax.scatter(
-                points[:, 0],
-                points[:, 1],
+            _scatter_projection(
```

Comment:
should we also add rasterized=rasterized, here?

Reply:
The old code does not have rasterized=rasterized here. I am not sure if I add some argument to ~2000 lines of code function "scatters" can cause any problem.